static/header.less

```diff
@@ -208,7 +208,7 @@
         background: @border-color;
         .dropdown-menu {
             display: block;
-            z-index:99;
+            z-index: @z-index-mid;
```

Comment:
This one doesn’t seem necessary either because the header already has a z-index.